fixtures/godox/sz300r.json

```diff
@@ -0,0 +1,371 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Godox SZ300R",
+  "categories": ["Dimmer", "Color Changer"],
+  "meta": {
+    "authors": ["Dyami Caliri"],
+    "createDate": "2024-05-17",
+    "lastModifyDate": "2024-05-17"
+  },
+  "links": {
+    "manual": [
+      "https://www.godox.com/static/upload/file/20230714/1689299183595059.pdf"
```

Comment:
I don't see a DMX chart in the manual for this fixture, either, but I see that all the channel data is identical to sl300r.json -- can we combine them?

Reply:
It's here: https://www.godox.com/static/upload/file/20230601/1685601290490750.pdf